test/model/TypeHierarhcy.test.ts

```diff
@@ -20,6 +28,15 @@ describe("TypeHierarchy", () => {
     hierarchy = new TypeHierarchy();
   });
 
+  describe("constructor", () => {
+    it("should build a TypeHierarchy from an EntityCollection", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)